user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Dashboards_app/Tutorials/Creating_a parameter_table_connected_to_an_element_feed.md

```diff
@@ -0,0 +1,111 @@
+
+1. Drag and drop the table visualization from the pane on the left to the dashboard.
+
+   This will add a table component, where the output of the GQI query will be shown.
+
+## Step 2: Configure the dropdown component to list all elements of a specific protocol
+
+1. From the data pane, drag the *Elements* data header onto the data input of the dropdown component.
+
+   ![](~/user-guide/images/Tutorial_Dropdown_Elements.png)
+
+1. From the data pane, drag the desired protocol onto the filter input of the dropdown component.
+
+   ![](~/user-guide/images/Tutorial_Dropdown_Protocol.png)
+
+The dropdown component will now list all available elements that match the protocol you selected.
+
+## Step 3: Configure a GQI query to display parameters from a specific protocol table and link it to the dropdown feed
+
+1. In the data pane, open the *QUERIES* section and click the "+" icon.
+
+   ![Add a query](~/user-guide/images/Tutorial_Add_Query.png)
+
+1. Enter a descriptive query name, e.g. *Disk Information Table*.
+
+1. Select the *Get parameters for elements where* data source.
+
+1. Set *Type* to "Protocol".
+
+1. Select a protocol.
+
+   > [!NOTE]
+   > This should be the same protocol as the one you used to filter the dropdown component in step 2.
+
+1. Select a protocol version.
+
+1. Select the protocol table that contains the data that you want to display in the table component.
+
+1. Optionally, add a *Select* operator to control which parameters from the specified protocol table to display in the GQI table.
+
+   > [!NOTE]
+   > It is important that *Element ID* column remains selected to allow linking to the dropdown component.
+
+1. Add a *Filter* operator to filter the query results using the element that will be selected in the dropdown feed.
+
+   1. Select the *Filter* operator.
+
+   1. Set *Column* to "Element ID".
+
+   1. Set *Filter method* to "regex".
+
+   1. In the *Value* box, click the *Link to feed* icon or open the *Link to feed* dialog.
+
+      1. Open the *Feed* box, and select the dropdown component.
+
+      1. Open the *Property* box, and select "Element ID".
+
+      1. Click *Apply*.
+
+1. Optionally, if you do not want the element ID to be displayed in the table, add another *Select* operator where "Element ID" is no longer selected.
+
+   Below, you can see an example of the query you made following the instructions above:
+
+   ![](~/user-guide/images/Tutorial_completed_query.png)
+
+1. Scroll up until you see the header of the query you are editing, and click the pen icon to save the query.
+
+1. Drag the query from the data pane onto the table component.
+
+   ![](~/user-guide/images/Tutorial_query_as_table_input.png)
+
+1. Click *Stop editing* to exit the edit mode.
+
+The table component should now display the parameters specified during query creation, and their values should change depending on the element selected in the dropdown feed.
+
```

Comment:
Could you add a screenshot of the final result at the end of the tutorial, so users get an idea of what their dashboard should look like when they're done?